tests/conftest.py

```diff
@@ -1,7 +1,15 @@
+import sys
+import os
+
 from pytest import fixture
 from pkg_resources import resource_filename
 from nchelpers import CFDataset
 
+# Add helpers directory to pythonpath: See https://stackoverflow.com/a/33515264
```

Comment:
Good grief, relative imports are confusing. There are dozens of stackoverflow questions about it, and few good answers. This was one answer I found a while ago. But after some thought, I was able to set things up for relative imports in the test directory. Phew.